nixos/modules/services/backup/restic.nix

```diff
@@ -196,6 +196,17 @@ in
           ];
         };
 
+        checkOpts = mkOption {
+          type = types.listOf types.str;
+          default = [ ];
+          description = ''
+            A list of options for 'restic check'.
+          '';
+          example = [
+            "--with-cache"
```

Comment:
Please add this to the tests so we make sure it is being used as expected.

Reply:
Hmm, I'm not sure how to go about getting this into a test. All of the `restic check` options I know of don't affect the outputs, they just are performance tuning parameters, so not sure how to test that.

Reply:
Couldn't you replace the binary of `restic` so it prints the command line?